Userland/Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -549,12 +550,11 @@ Optional<Value> string_to_number(StringView string)
         return Value(bigint.to_double());
     }
 
-    char* endptr;
-    auto parsed_double = strtod(text.characters(), &endptr);
-    if (*endptr)
+    auto parsed_double = text.to_double(AK::TrimWhitespace::No);
```

Comment:
Oh god, another name for the same thing haha - feel free to ignore the previous comment and call that `parsed_double` as well